lib/rules/custom-property-no-missing-var-function/__tests__/index.mjs

```diff
@@ -32,6 +32,10 @@ testRule({
 			code: 'a { color: var(--foo, red); }',
 			description: 'undeclared dashed-idents in var with fallback',
 		},
+		{
+			code: 'a { color: var(--foo, var(--bar)); }',
```

Comment:
It's possible for fallback values to also have fallbacks so would be good to have a test for that as well to ensure they're recursively tested, eg:

```css 
a { color: var(--foo, var(--bar, var(--foobar, #000))); }
```

Reply:
Thanks! I applied your suggestion.
[chore: add test for nested fallbacks](https://github.com/stylelint/stylelint/pull/8116/commits/b459db880c16a8525c7b3de41da85ec5c1af8b86)